account_payment_transfer_reconcile_batch/wizard/account_payment_line_create.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, models
+
+
+class PaymentOrderCreate(models.TransientModel):
+    _inherit = 'account.payment.line.create'
+
+    @api.multi
+    def _prepare_move_line_domain(self):
+        domain = super(PaymentOrderCreate, self)._prepare_move_line_domain()
```

Comment:
You can test this creating a new payment order and seeing that the existing invoice is not added again to the new payment order.